### ISSUE TITLE
Screen reader doesn't announces "Searching..." message when switching tabs

### ISSUE THREAD

**Issue by dominoFire** [NuGet/Home#9307]
Link: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/994460/
